torch/csrc/stable/utils.h

```diff
@@ -0,0 +1,342 @@
+#pragma once
```

Comment:
I firmly dislike naming things "utils" because it is synonymous with "stuff" and helps neither predict their current contents nor limit their future contents. Instead I would consider a specific name, like say StableIValueConversions.h .

Reply:
renamed!!!!!! stableivalue_conversions.h